admin_guide/build_defaults_overrides.adoc

```diff
@@ -0,0 +1,78 @@
+= Setting Build Defaults and Overrides
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The BuildDefaults admission control plugin allows an administrator to set global defaults for settings such as the git
+HTTP and HTTPS proxy, and default envrionment variables. These defaults will not overwrite values that have been configured
+for a particular build. However, if they are not present, they will be set as the default value.
```

Comment:
"if they are not present on the build definition" (otherwise it's ambiguous as to whether "not present" refers to not present in the cluster config or not present in the build)